### PR TITLE
Setup integration with Scrutinizer

### PR DESCRIPTION
**Pull request by hgraca**
Integrate the repo with scrutinizer CI so we are sure everything is OK before merging.

### CODE REVIEW

File: tests/phpunit.bootstrap.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?php
 
-(function(): void {
+(function (): void {
     require_once __DIR__ . '/../vendor/autoload.php';
-    $secondsToSleep = 3;
+    $secondsToSleep = 30;
```
Comment (malukenho) [inline tests/phpunit.bootstrap.php]: May we can set an ENV var via `phpunit.xml` for that?

Comment (hgraca) [reply tests/phpunit.bootstrap.php]: Im still testing with it, This last commit will be removed when everything is ok.

At the moment theres an issue with running Docker inside scrutinizer. The container where the PHP code runs can not reach the container with rabbitmq... so the tests fail. :(

I already sent a message to them, and Im waiting for some feedback.

Comment (malukenho) [reply tests/phpunit.bootstrap.php]: Ouch! That's okay then :/

File: tests/container/docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: '3.2'
```
Comment (jeroenvdheuvel) [inline tests/container/docker-compose.yml]: Why did we go down in minor version? Just curious

**Pull request merged**